versions.txt

```diff
@@ -6,4 +6,7 @@ operator=1.2.0
 
 # Represents the current release of ADOT language instrumentation.
 aws-otel-java-instrumentation=v1.32.1
-aws-otel-python-instrumentation=v0.0.1
\ No newline at end of file
+aws-otel-python-instrumentation=v0.0.1
+
+dcgm-exporter=3.3.3-3.3.1-ubuntu22.04
```

Comment:
`3.3.3-3.3.1-ubuntu22.04` is this a typo?

Reply:
it's the actual version https://github.com/NVIDIA/dcgm-exporter/releases/tag/3.3.3-3.3.1